File: CS-Examples/17_HeaderAndFooter/AdjustHeaderFooterHeight.cs
using System;
using System.Windows.Forms;
using Spire.Doc;

namespace AdjustHeaderFooterHeight
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            string input = @"..\..\..\..\..\..\Data\HeaderAndFooter.docx";

			//Create a word document
			Document doc = new Document();

			//Load the file from disk
			doc.LoadFromFile(input);

			//Get the first section
			Section section = doc.Sections[0];

			//Adjust the height of headers in the section
			section.PageSetup.HeaderDistance = 100;

			//Adjust the height of footers in the section
			section.PageSetup.FooterDistance = 100;

			//Save the document
			string output = "AdjustHeaderFooterHeight.docx";
			doc.SaveToFile(output, FileFormat.Docx);

			// Dispose the document
			doc.Dispose();

            Viewer(output);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/17_HeaderAndFooter/AdjustHeaderFooterHeight.vb
Imports Spire.Doc

Namespace AdjustHeaderFooterHeight
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim input As String = "..\..\..\..\..\..\Data\HeaderAndFooter.docx"

			'Create a word document
			Dim doc As New Document()

			'Load the file from disk
			doc.LoadFromFile(input)

			'Get the first section
			Dim section As Section = doc.Sections(0)

			'Adjust the height of headers in the section
			section.PageSetup.HeaderDistance = 100

			'Adjust the height of footers in the section
			section.PageSetup.FooterDistance = 100

			'Save and launch document
			Dim output As String = "AdjustHeaderFooterHeight.docx"
			doc.SaveToFile(output, FileFormat.Docx)

			'Dispose the document
			doc.Dispose()
			
			Viewer(output)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
